cmake/std/atdm/spack-rhel6/environment.sh

```diff
@@ -131,11 +131,13 @@ export ATDM_CONFIG_NETCDF_LIBS="-L${BOOST_ROOT}/lib;-L${NETCDF_ROOT}/lib;-L${NET
 # NOTE: SEMS does not provide the correct *.so files for NetCDF so we can't
 # use them in a shared lib build :-(
 
-export ATDM_CONFIG_BINUTILS_LIBS="${BINUTILS_ROOT}/lib/libbfd.so;${BINUTILS_ROOT}/lib64/libiberty.a"
+export ATDM_CONFIG_BINUTILS_LIBS="${BINUTILS_ROOT}/lib/libbfd.so;${BINUTILS_ROOT}/lib64/libiberty.a;${GETTEXT_ROOT}/lib/libintl.a;${LIBICONV_ROOT}/lib/libiconv.so"
 # NOTE: Above, we have to explicitly set the libs to use libbdf.so instead of
 # libbdf.a because the former works and the latter does not and TriBITS is set
 # up to only find static libs by default!
 unset BINUTILS_ROOT
+# NOTE: Above, you have to unset this or the configure dues in in the compiler
+# check.  What the heck?
```

Comment:
Did you mean to write this?

Reply:
oops, I didn't mean to hit approve; just cancelled

Reply:
Just pushed commit 5410b315ba738ea071ce754e31b11b1a873b5e4f that hopefully improves this comment.  Please let me know.  Thanks!